src/test/java/core/basesyntax/service/RegistrationServiceTest.java

```diff
@@ -0,0 +1,132 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceTest {
+    private static StorageDao defaultStorage;
+    private static RegistrationServiceImpl registrationService;
+
+    @BeforeAll
+    static void beforeAll() {
+        defaultStorage = new StorageDaoImpl();
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        User firstUser = new User();
+        firstUser.setId(1L);
+        firstUser.setLogin("first");
+        firstUser.setPassword("111111");
+        firstUser.setAge(18);
+        User secondUser = new User();
+        secondUser.setId(2L);
+        secondUser.setLogin("second");
+        secondUser.setPassword("222222");
+        secondUser.setAge(19);
+        User thirdUser = new User();
+        thirdUser.setId(3L);
+        thirdUser.setLogin("third");
+        thirdUser.setPassword("333333");
+        thirdUser.setAge(20);
+        Storage.people.add(0, firstUser);
+        Storage.people.add(1, secondUser);
+        Storage.people.add(2, thirdUser);
+    }
+
+    @Test
+    void registerValidUser_ok() {
```

Comment:
```suggestion
    void register_validUser_ok() {
```

There are a lot of ways to name your test methods. The main point is that they should have informative names and be consistent along with other developers in your team. For this task use such convention: `<methodUnderTest>_<state>_<expectedBehavior>`; For example, if we are testing the method register with a `null` user's age the test method name should be `register_nullAge_notOk`. `notOk` is because the test expects the register method to throw an exception.


fix naming in all tests